hack/release-tool.py

```
@@ -158,6 +158,19 @@ def create_releasepayload_patch(action, custom_reason):
 
 
+def resolve_imagestream_from_releasepayload(ctx, namespace, release):
+    """Look up the imagestream name from the ReleasePayload's spec.payloadCoordinates.imagestreamName."""
+    with oc.options(ctx), oc.tracking(), oc.timeout(15):
+        try:
+            with oc.project(namespace):
+                payload = oc.selector(f'releasepayload/{release}').object(ignore_not_found=True)
+                if not payload:
+                    return None
+                return payload.model.spec.payloadCoordinates.imagestreamName
+        except (ValueError, OpenShiftPythonException, Exception):
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)